mqtt/mqtt-bridge/src/bridge.rs

```diff
@@ -127,13 +131,15 @@ where
     S: StreamWakeableState + Send,
 {
     pub async fn run(self) -> Result<(), BridgeError> {
-        info!("Starting {} bridge...", self.connection_settings.name());
```

Comment:
We will have to make sure to log names for other bridges when we support them.

Reply:
Yeah, we log them in the span.